pages/docs/guides/rate-limiting.mdx

```diff
@@ -0,0 +1,103 @@
+export const description = 'Prevent excessive function runs over a given time period by skipping events beyond a specific limit. Ideal for protecting against abuse.'
+
+# Rate limiting
+
+Rate limiting is a _hard limit_ on how many function runs can start within a time period. Events that exceed the rate limit are _skipped_ and do not trigger functions to start. This prevents excessive function runs over a given time period. Some use cases for rate limiting include:
+
+* Preventing abuse of your system
+* Reducing frequency of data synchronization functions
+* Smoothing out noisy [webhook](/docs/platform/webhooks) events
```

Comment:
Not sure what "Smoothing out" means here. Would you have a common or Plain English expression to offer here?